src/app/api/user/[id]/route.ts

```diff
@@ -23,7 +22,21 @@ export async function GET(
       return new Response(`No User found for id: ${id}`, { status: 404 });
     }
 
-    return NextResponse.json(user);
+    const requiredFields = [
+      "id",
+      "firstName",
+      "lastName",
+      "email",
+      "member",
+      "verified",
+      "remainingSessions",
+    ];
+
+    const filteredUser = Object.fromEntries(
+      Object.entries(user).filter(([key]) => requiredFields.includes(key)),
+    );
+
```

Comment:
instead of doing this you can specify what columns you want to retrieve in the db query.
https://orm.drizzle.team/docs/rqb#partial-fields-select